src/utils/searchUtils.ts

```diff
@@ -58,18 +59,29 @@ export function getSearchResultsHeading(
 }
 
 function buildQueryDisplayString(searchParams: SearchParams): string {
-  const searchFields = textInputFields.concat([
-    { name: "journal_title", label: "Journal Title" },
-    { name: "standard_number", label: "Standard Number" },
-    { name: "creatorLiteral", label: "author" },
-  ])
+  const searchFields = advSearchFields
+    // Lowercase the adv search field labels:
+    .map((field) => ({ ...field, label: field.label.toLowerCase() }))
+    .concat([
+      { name: "journal_title", label: "journal title" },
+      { name: "standard_number", label: "standard number" },
+      { name: "creatorLiteral", label: "author" },
+      { name: "oclc", label: "OCLC" },
```

Comment:
When would we be displaying these identifiers besides oclc? Or is this a carryover from DFE?